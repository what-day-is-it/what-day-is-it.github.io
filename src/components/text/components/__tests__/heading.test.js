import React from 'react';
import { shallow } from 'enzyme';
import Heading from '../heading';

describe('Heading', () => {
  const props = {
    level: 5,
    children: <div>Heading</div>,
    className: 'foobar'
  };
  const wrapper = shallow(<Heading {...props} />);
  const noPropsWrapper = shallow(<Heading />);

  it('should render', () => {
    expect(wrapper).toMatchSnapshot();
    expect(noPropsWrapper).toMatchSnapshot();
  });

  it('should render the props', () => {
    expect(wrapper.find('.foobar').exists()).toEqual(true);
    expect(wrapper.find('.heading-5').exists()).toEqual(true);
  });
});
