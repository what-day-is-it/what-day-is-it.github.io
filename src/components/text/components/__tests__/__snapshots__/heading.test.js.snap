// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading should render 1`] = `
<h5
  className="
        heading-5 black
        weight-normal align-undefined
        
        font-undefined undefined
        foobar
      "
>
  <div>
    Heading
  </div>
</h5>
`;

exports[`Heading should render 2`] = `
<h2
  className="
        heading-2 black
        weight-normal align-undefined
        
        font-undefined undefined
        
      "
/>
`;
